app/views/dashboard.view.php

```diff
@@ -0,0 +1,128 @@
+<link rel="stylesheet" href="<?=ROOT?>/assets/css/global.css">
+<link rel="stylesheet" href="<?=ROOT?>/assets/css/dashboard.css">
+<link rel="stylesheet" href="<?=ROOT?>/assets/css/home.css">
+<script src="<?=ROOT?>/assets/js/dashboard.js" defer></script>
+ 
+<div class="navbar">
+    <div class="profile">
+        <img src="<?= ROOT ?>/assets/images/profile.png" alt="">
+        <p><?= $_SESSION['user']->email; ?></p>
+    </div>
+    
+    <ul>
+        <li><a href="<?=ROOT?>/home">Home</a></li>
+        <?php if ($_SESSION['user']->role_id == 2): ?>
+            <li><a href="<?=ROOT?>/manageUsers">Manage Users</a></li>
+        <?php endif; ?>
+        <li><a href="<?=ROOT?>/signout">Sign Out</a></li>
+    </ul>
+</div>
+
+<div class="add">
+    <a href="<?=ROOT?>/insertPost">ADD NEW POST</a>
+</div>
+
+<?php if (!empty($data['posts'])): ?>
+    <?php foreach ($data['posts'] as $post): ?> 
+        <div class="post">
+            <?php if ($post->user_id == $_SESSION['user']->id || $_SESSION['user']->role_id == 2): ?>
+                <div class="operations">
+                    <?php if ($_SESSION['user']->role_id != 2 || $post->user_id == $_SESSION['user']->id): ?>
```

Comment:
This is example of "magic numbers" . Number 2 should be stored in the constant, probably in the model User or even better Role.php `public const ROLES = ['admin' => 2]`